src/graphql/tidb.rs

```diff
@@ -250,11 +250,13 @@ pub struct TidbOutput {
     version: String,
 }
 
+#[allow(dead_code)]
 struct TidbTotalCount;
 
 #[Object]
 impl TidbTotalCount {
     /// The total number of edges.
+    #[allow(dead_code)]
```

Comment:
@danbi2990,

When a part of the code is truly unused, removing it is better than silencing Clippy warnings. `TotalCount` is usually used to provide the total number of items for pagination, and the fact that this code is unused suggests that no pagination is applied to `Tidb`.

I suspect that the person who wrote this part originally intended to implement pagination but ended up not doing so. Alternatively, they might have mistakenly copied this block even though it wasn’t necessary.

Simply silencing the warning makes it harder to catch this problem. On the other hand, removing the block would make it clearer to other contributors that `Tidb` does not have pagination.

I will remove this block myself.